index.js

```diff
@@ -20,17 +20,17 @@ module.exports = function (str, cb) {
 	});
 };
 
-module.exports.sync = function (str) {
-	return zlib.gzipSync(str, opts).length;
+module.exports.sync = function (str, opts) {
+	return zlib.gzipSync(str, opts || defaultOpts).length;
```

Comment:
Better to use `Object.assign` so it's possible to add more default options in the future.